### ISSUE TITLE
How to update Nipple JS container after device orientation change?

### ISSUE THREAD

**Issue by that-ben** [yoannmoinet/nipplejs#45]
Hi, thank you so much for your Nipple JS efforts.  This plugin is AMAZING and really a life saver when it comes to all these iOS updates breaking other methods of implementing DPAD and joystick controllers.  I found Nipple JS to be the BEST javascript plugin out there and it works universally on every device, which is totally impressive.

ISSUE/QUESTION: I have my Nipple JS created in a HTML element that is POSITION:FIXED; and it works really good, except when I change my device orientation, it then needs its coordinates adjusted, or else the Nipple JS events launch, but the coordinates it returns are offset by a lot.  Is there a clean way to tell Nipple JS to update its container coordinates? Or should I unbind it and recreate it everytime (I hope not!)

Thanks!

**Comment by yoannmoinet** [comment]
Hi !
Thank you for those kind words 😜 

For the rotation, there is some code in the library that should handle that for you.
I might have missed some native updates I guess, you can take a [look there](https://github.com/yoannmoinet/nipplejs/blob/master/src/manager.js#L18), it might enlighten what's happening in your case.

I'm due for a large update of the lib, as it becomes unstable in more and more cases, and pass it to a more maintainable architecture (ES6, modules and stuffs).
But I'm currently out of time.

Please, keep me updated with your investigation.

**Comment by that-ben** [comment]
Thank you so much for replying.  I have looked at the event handler you showed me and altough it looks very straightforward, I couldn't find why it does that on my iPhone 7 iOS 10.3 and not on my iPhone 5 iOS 10.1.  Either way, for now, I have fixed my implementation by removing every position:fixed and adapted them to position:absolute instead.  This very small change completely solved the case for me.

Thanks
